api/handlers/changes.js

```diff
@@ -41,90 +37,68 @@ var getDepth = function(userCtx) {
   return depth;
 };
 
-var bindSubjectIds = function(feed, callback) {
-  var startTime = startTimer();
-
-  auth.getFacilityId(feed.req, feed.userCtx, function(err, facilityId) {
+const getSubjectIds = (facilityId, userCtx, callback) => {
```

Comment:
All the `bindXYZ` functions got renamed to `getXYZ` which makes it a little easier to follow. I also refactored them so they only take the specific properties as parameters rather than the whole `feed` object.